react/features/invite/components/DialInNumbersForm.js

```diff
@@ -204,11 +208,18 @@ class DialInNumbersForm extends Component {
     _createDropdownTrigger(triggerText) {
         return (
             <div className = 'dial-in-numbers-trigger'>
-                <input
-                    className = 'input-control'
-                    readOnly = { true }
-                    type = 'text'
-                    value = { triggerText || '' } />
+                <div className = 'form-control__input-container'>
+                    <FieldText
+                        compact = { true }
+                        isLabelHidden = { true }
+                        isReadOnly = { true }
+                        label = 'Select Dial-In Number'
+                        onChange = { _onNoop }
```

Comment:
The name "onNoop" sounds a little bit weird here. In general the shortcut with using directly noop doesn't help to understand what behaviour we need to cancel when "onChange". Looking at this code one may even decide that removing onChange property would work the same way. It could be better to just create a private function that also has some documentation explaining why it's important to return undefined on FieldText.onChange.